client/src/charts/progressGauge.js

```diff
@@ -0,0 +1,40 @@
+import "./progressGauge.scss";
+import { successDarkColor } from "../core/color_defs";
+import { Format } from "../components";
+
+export default class ProgressGauge extends React.Component {
+  render() {
+    const { value, total_value, finished_color } = this.props;
+    const percentage = value / total_value;
+
+    const progress = _.times(total_value, (i) => {
+      if (i < value) {
+        return (
+          <div
+            key={i}
+            className="progress-circle"
+            style={{
+              backgroundColor: finished_color || successDarkColor,
+            }}
+          >
+            <p />
+          </div>
+        );
+      } else {
+        return <div key={i} className="progress-circle empty" />;
+      }
+    });
+
+    return (
+      <div>
+        <h4 style={{ textAlign: "center" }}>
+          {value} / {total_value}
+        </h4>
+        <div>{progress}</div>
```

Comment:
How about this? I think this is cleaner and elegant without the need of if statement
```
const filled_circle_style= {backgroundColor: finished_color || successDarkColor}
const empty_circle_style = {border: "2px solid"}
return <div>
  {_.times(total_value, (i) => 
    <div
      className="progress-circle"
      key={i}
      style={i < value ? filled_circle_style : empty_circle_style}
    />
  )}
// ...and other elements
</div>
```